cmd/legacy_main.go

```diff
@@ -375,8 +375,15 @@ func Mount(newConfig *cfg.Config, bucketName, mountPoint string) (err error) {
 		// programme is running as daemon process.
 		env = append(env, fmt.Sprintf("%s=true", logger.GCSFuseInBackgroundMode))
 
+		var logFile *os.File
+		if newConfig.Logging.FilePath != "" {
+			logFile, err = os.OpenFile(string(newConfig.Logging.FilePath), os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0644)
+			if err != nil {
+				return err
+			}
+		}
 		// Run.
-		err = daemonize.Run(path, args, env, os.Stdout)
+		err = daemonize.Run(path, args, env, os.Stdout, logFile)
```

Comment:
Should we redirect it to os.Stderr instead of creating a new file handle?

Reply:
We can't redirect it to os.Stderr because if the user closes the terminal after mounting, the error messages will be lost.